gap/action/orbstab.gi

```diff
@@ -589,9 +589,9 @@ BindGlobal( "OrbitIntegralAction", function( G, mats, e, f )
         return rec( stab := StabilizerIntegralAction(G, mats, e),
                     prei := One( G ) );
     fi;
-    if RankMat( [e,f] ) = 1 or ForAll( mats, x -> e*x = e) then
-        return false;
-    fi;
+    # if RankMat( [e,f] ) = 1 or ForAll( mats, x -> e*x = e) then
+    #     return false;
+    # fi;
```

Comment:
This change comes from the commit "Fixed bug in orbits".

What is that bug? Do we have a test case that reproduces the bug and verifies the fix works? Then it should be added in a .tst file.

Also if this code is wrong, can we just delete it?
```suggestion
```

Alternatively, if this is meant to be a reminder about a feature that should be re-enabled later, then there should be a comment explaining this.